lib/install/management/appliance.go

```diff
@@ -142,7 +141,7 @@ func (d *Dispatcher) checkExistence(conf *config.VirtualContainerHostConfigSpec,
 		verr = errors.Errorf("Found virtual machine %q, but it is not a VCH. Please choose a different virtual app.", conf.Name)
 		return verr
 	}
-	err = errors.Errorf("Virtual app %q already exists. Please delete it before reinstalling.", conf.Name)
+	err = errors.Errorf("A VCH with the name %q already exists. Please delete it or choose a different VCH name before attempting another reinstall", conf.Name)
```

Comment:
I'd reword to the following:

_A VCH with the name %q already exists. Please choose a different name before attempting another install_

I think the user can determine if they want to delete vs. rename...